code/modules/surgery/amputation.dm

```diff
@@ -7,6 +7,12 @@
 	target_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
 	possible_locs = list(BODY_ZONE_R_ARM, BODY_ZONE_L_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG, BODY_ZONE_HEAD)
 
+/datum/surgery/amputation/can_start(mob/user, mob/living/carbon/target)
+	if(isshadowperson(target)) // no cutting off shadow limbs because they look horrendous when spliced with other species
+		to_chat(user, span_warning("[target]'s body is made out of shadows!"))
```

Comment:
`can_start` decides whether the surgery shows up as an option in the radial menu, there shouldn't be a `to_chat` here.